wrangler-api/src/main/java/io/cdap/wrangler/api/lineage/Mutation.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2017-2019 Cask Data, Inc.
+ * Copyright © 2019 Cask Data, Inc.
```

Comment:
This should still be 2017-2019 since the file is not added in this pr

Reply:
fixed